pepper-apis/src/main/java/org/broadinstitute/ddp/security/AesUtil.java

```diff
@@ -160,23 +171,34 @@ public static void main(String[] args) throws Exception {
                 return;
             }
             TransactionWrapper.withTxn(handle -> {
-                List<Map<String, Object>> clientList = handle.createQuery(
-                        "select auth0_client_id, auth0_signing_secret from client").mapToMap().list();
+                List<Map<String, Object>> clientList = handle.createQuery(SELECT_CLIENT_TENANT_AND_SIGNING_SECRET_QUERY)
+                        .mapToMap()
+                        .list();
                 for (Map<String, Object> client : clientList) {
                     String auth0Secret = (String) client.get("auth0_signing_secret");
-                    String auth0ClientId = (String) client.get("auth0_client_id");
-                    System.out.println("auth0clientId = " + auth0ClientId);
+                    String auth0ClientIdTenantIdStr = (String) client.get("auth0_client_id_tenant_id");
+                    ClientIdTenantId clientIdTenantId = new ClientIdTenantId(auth0ClientIdTenantIdStr);
```

Comment:
Instead of concat-then-split by hand, you can have JDBI do it, just like how we do it for most other DAOs stuff. I don't remember the method names off-the-top of my head, but you can use their fluent API something like `handle.createQuery(..).registerConstructorMapper(..).mapTo(ClientIdTenantId.class).list()`. Note, I think you have to make this class `public` so JDBI can access it via reflection.

Reply:
Done.